BrowserServicesKit/Sources/Subscription/SubscriptionURL.swift

```diff
@@ -24,13 +24,13 @@ public enum SubscriptionURL {
 
     case baseURL
     case purchase
+    case welcome
     case faq
-    case activateViaEmail
-    case addEmail
```

Comment:
The `addEmail` and `addEmailToSubscriptionSuccess` are removed since add email URLs are no longer used directly - add email is part of the new activation flow. These also are only used on iOS where we check for what page is displayed to adjust the UI (navbar and back button behavior), they are defined there directly.